### ISSUE TITLE
Pip bug: Cannot import submodules on runtime

### ISSUE THREAD

**Issue by bdashore3** [Dan-wanna-M/formatron#9]
Hi there! I was trying to use this package with tabbyAPI and I get a traceback stating the following:

```
Traceback (most recent call last):
  File "D:\tabbyAPI\main.py", line 22, in <module>
    from formatron.grammar_generators.json_generator import JsonGenerator
  File "D:\tabbyAPI\venv\Lib\site-packages\formatron\grammar_generators\json_generator.py", line 9, in <module>
    import extractor
ModuleNotFoundError: No module named 'extractor'
```

Reproduction:
1. Install the package: `pip install formatron`
2. Try importing the JsonGenerator via `from formatron.grammar_generators.json_generator import JsonGenerator`
3. Error shows when starting the program

**Comment by Dan-wanna-M** [comment]
Gonna investigate this

**Comment by Dan-wanna-M** [comment]
Should be fixed in `v0.3.1` now, could you verify it?

**Comment by bdashore3** [comment]
Works, thanks!
